types/lightship/index.d.ts

```diff
@@ -10,30 +10,57 @@ import { Server } from 'http';
  * A teardown function called when shutdown is initialized.
  * @param userConfiguration User configuration object
  */
-export function createLightship(userConfiguration?: UserConfigurationType): LightshipType;
+export function createLightship(configuration?: ConfigurationInputType): LightshipType;
```

Comment:
Renaming this to match the official type names in the [Usage instructions](https://github.com/gajus/lightship#usage) and [Flow Types](https://github.com/gajus/lightship/blob/master/src/types.js).